consistency in packet names (#293)
QuiltMC/quilt-mappings: CONVENTIONS.md

Name screen coordinates `x` and `y`, rather than `left` and `top`.

### Network Packets

Always suffix packet names with `S2CPacket` (server to client) or `C2SPacket` (client to server).

> Packet names should be in the form of `[Noun][Verb]`: `BlockUpdateS2CPacket`, `PlayerMoveC2SPacket`, etc., with the verb conjugated in the infinitive form.

Reviewer: I don't think requiring infinitive form for every packet really makes sense. For example `InventoryItemPick(C2SPacket)` makes more sense in past form (`InventoryItemPicked`), while `EntityNbtQuery(C2SPacket)` is fine as-is

Author: I specified that this is a loose convention, and should not be followed strictly (given that I've already failed to do that :p)

Name screen coordinates `x` and `y`, rather than `left` and `top`.

### Network Packets

Always suffix packet names with `S2CPacket` (server to client) or `C2SPacket` (client to server).

> Packet names should be in the form of `NounVerbed`: `ResourcePackSentS2CPacket`, `BlockUpdatedS2CPacket`, `PlayerMovedC2SPacket`, etc.

Reviewer 2: ```suggestion
Packet names should be in the form of `[Noun][Verbed]`: `ResourcePackSentS2CPacket`, `BlockUpdatedS2CPacket`, `PlayerMovedC2SPacket`, etc.
```

> In some cases, the suffix `WithX` can come after the verb. This is to prevent situations where an awkward `And` must be inserted, such as `PlayerAndEntityInteractedC2SPacket`, over `PlayerInteractedWithEntityC2SPacket`.

Reviewer 2: ```suggestion
In some cases, the suffix `WithX` can come after the verb. This is to prevent situations where an awkward `And` would be inserted, avoiding `PlayerAndEntityInteractedC2SPacket`, in favor of `PlayerInteractedWithEntityC2SPacket`.
```

> Sometimes, a packet is purely data and does not have an action associated. In this case the packet name should simply be `NounSentS2CPacket` or `NounSentC2SPacket`.

Reviewer 2: ```suggestion
Sometimes, a packet is purely data and does not have an action associated. In this case the packet name should simply be `[Noun]SentS2CPacket` or `[Noun]SentC2SPacket`.
```

> The listener (found in the `net.minecraft.network.listener` package) should be oppositely named: `onNounReceived`.

Reviewer 2: ```suggestion
The listener (found in the `net.minecraft.network.listener` package) should be oppositely named: `on[Noun]Received`.
```
